catalyst/core/callbacks/metrics.py

```diff
@@ -50,38 +120,39 @@ def __init__(
         list_args: List,
         input_key: str = "targets",
         output_key: str = "logits",
-        **metric_params,
+        multiplier: float = 1.0,
+        **metrics_kwargs,
     ):
-        super().__init__(CallbackOrder.Metric)
-        self.prefix = prefix
-        self.metric_fn = partial(metric_fn, **metric_params)
+        super().__init__(
+            prefix=prefix,
+            metric_fn=metric_fn,
+            input_key=input_key,
+            output_key=output_key,
+            multiplier=multiplier,
+            **metrics_kwargs,
+        )
         self.list_args = list_args
-        self.input_key = input_key
-        self.output_key = output_key
-
-    def on_batch_end(self, state: "_State"):
-        outputs = state.batch_out[self.output_key]
-        targets = state.batch_in[self.input_key]
 
-        metrics_ = self.metric_fn(outputs, targets, self.list_args)
+    def on_batch_end(self, state: _State):
+        metrics_ = self._compute_metric(state)
 
         for arg, metric in zip(self.list_args, metrics_):
             if isinstance(arg, int):
                 key = f"{self.prefix}{arg:02}"
             else:
                 key = f"{self.prefix}_{arg}"
-            state.batch_metrics[key] = metric
+            state.batch_metrics[key] = metric * self.multiplier
 
 
-class MetricAggregatorCallback(Callback):
+class MetricAggregationCallback(Callback):
     """
     A callback to aggregate several metrics in one value.
     """
     def __init__(
         self,
         prefix: str,
         metrics: Union[str, List[str], Dict[str, float]] = None,
-        mode: str = "sum",
+        mode: str = "mean",
```

Comment:
why it has changed?
if we really want to change the default, I think we should raise some alarm warning as it will make the configs from previous versions incompatible

Reply:
hmm, as the key also changed (`aggregation_fn` -> `mode`) it's probably OK right now, but still can be misleading